<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PhotoSort42</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit_photoInput">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>5</y>
      <width>291</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sort">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>start sort mechanism</string>
    </property>
    <property name="text">
     <string>sort</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_compareInput">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>135</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_output">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>216</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Select a sort mechanism for your photos</string>
    </property>
    <property name="title">
     <string>sort mechanism</string>
    </property>
    <widget class="QCheckBox" name="checkBox_landscape">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>291</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>landscape photos</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_onePerson">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>281</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>photo of single person</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_group">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>group photo</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_photoInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>Select photos from a folder to sort. You have to choose at least 2 photos.</string>
    </property>
    <property name="text">
     <string>photos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_compareInput">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="font">
     <font>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="toolTip">
     <string>Choose a comparative picture of a person for the sort mechanism</string>
    </property>
    <property name="text">
     <string>comparative picture</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_compareImage">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>111</width>
      <height>141</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_photoOutput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Select an output folder for the result</string>
    </property>
    <property name="text">
     <string>output</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>637</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPhotoSort42">
    <property name="title">
     <string>PhotoSort42</string>
    </property>
   </widget>
   <addaction name="menuPhotoSort42"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
